package.json

```diff
@@ -25,137 +25,149 @@
     "locales:missing": "node tools/findMissingLocales"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.0.0-beta.54",
+    "@babel/polyfill": "^7.0.0",
     "classnames": "^2.2.6",
     "detect-browser": "^3.0.0",
-    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-node": "^7.0.1",
-    "eslint-plugin-promise": "^3.8.0",
+    "eslint-plugin-promise": "^4.0.0",
     "font-awesome": "^4.7.0",
     "hashids": "^1.1.4",
     "history": "^4.7.2",
-    "i18next": "^11.3.6",
-    "i18next-browser-languagedetector": "^2.2.2",
+    "i18next": "^11.6.0",
+    "i18next-browser-languagedetector": "^2.2.3",
     "lodash": "^4.17.10",
     "moment": "^2.22.2",
     "moment-timezone": "^0.5.21",
     "normalize-path": "^3.0.0",
     "normalize.css": "^8.0.0",
     "query-string": "^6.1.0",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
-    "react-dropdown": "^1.5.0",
+    "react": "^16.4.2",
+    "react-dom": "^16.4.2",
+    "react-dropdown": "^1.6.2",
     "react-fontawesome": "^1.6.1",
     "react-helmet": "^5.2.0",
     "react-highlighter": "^0.4.2",
-    "react-i18next": "^7.8.1",
+    "react-i18next": "^7.11.0",
     "react-onclickoutside": "^6.7.1",
     "react-redux": "^5.0.7",
     "react-spinkit": "^3.0.0",
     "react-styled-flexboxgrid": "^2.4.0",
     "react-tooltip": "^3.6.1",
-    "recompose": "^0.27.1",
+    "recompose": "^0.29.0",
     "redux": "^4.0.0",
-    "redux-actions": "^2.4.0",
+    "redux-actions": "^2.6.1",
     "redux-first-router": "^0.0.20-next",
     "redux-first-router-link": "^1.4.2",
     "redux-logger": "^3.0.6",
-    "redux-responsive": "^4.3.7",
+    "redux-responsive": "^4.3.8",
     "redux-thunk": "^2.3.0",
-    "styled-components": "^3.3.3",
+    "styled-components": "^3.4.5",
     "whatwg-fetch": "^2.0.4"
   },
   "devDependencies": {
-    "@babel/cli": "^7.0.0-beta.54",
-    "@babel/core": "^7.0.0-beta.54",
-    "@babel/plugin-transform-react-jsx": "^7.0.0-beta.54",
-    "@babel/plugin-transform-runtime": "^7.0.0-beta.54",
-    "@babel/preset-env": "^7.0.0-beta.54",
-    "@babel/preset-flow": "^7.0.0-beta.54",
-    "@babel/preset-react": "^7.0.0-beta.54",
-    "@babel/preset-stage-1": "^7.0.0-beta.54",
-    "@babel/runtime": "^7.0.0-beta.54",
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
```

Comment:
Why are all these proposals needed? A change in babel 7?

Reply:
Yes, they no longer provide the stage-1 preset (or any stage-x) instead we should explicitly define the proposals we want to use. We could even define our own babel-preset